packages/Takeover/src/Takeover.js

```diff
@@ -37,7 +39,10 @@ const defaultProps = {
   zIndex: zValue(5),
 };
 
-const Takeover = props => {
+const TakeoverOverlay = Overlay;
+delete TakeoverOverlay.propTypes.isOpen;
```

Comment:
LGTM, just curious is it good to do something like this? isn't more clear to define all propTypes again in this case for consumers? or define a common prop type and reuse? for example:
```
const BasicOverlayPropTypes = {
  isOpen:PropTypes.bool,
  children: PropTypes.func,
  ...
}
```
and use like:

```
Overlay.propTypes.= {
  ...BasicOverlayPropTypes,
 isOpen: PropTypes.bool.isRequired
}

TakeoverOverlay.propTypes.= BasicOverlayPropTypes
```

Reply:
Sorry, I didn't notice this comment earlier! I think this is `basicPropTypes` composition approach makes sense in some cases, but here I'm just using the `propTypes` for a prop collector, it's not even a real, functional component, so the extra complexity added to `<Overlay>` (which shouldn't really care about how `<Takeover>` is using it) feels unnecessary.